model/src/main/scala/org/broadinstitute/dsde/rawls/model/GoogleProjectModel.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.dsde.rawls.model
+
+import spray.json.{DefaultJsonProtocol, RootJsonFormat}
+import org.broadinstitute.dsde.rawls.model.UserModelJsonSupport._
+
+object GoogleProjectModel {
+  def newGoogleProject(googleProjectId: String,
+                       billingAccount: Option[String],
+                       message: Option[String],
+                       billingProjectId: String
+  ) = RawlsGoogleProject(googleProjectId, billingAccount, message, RawlsBillingProjectName(billingProjectId))
+
+}
+
+//TODO should any other fields be objects instead of strings?
```

Comment:
you can use `GoogleProjectId` and `RawlsBillingAccountName` for type safety

Also, does it make sense to move these models closer to other models? You could put these in UserAuth, for example